src/lib/pages/library/ui/library-page.svelte

```diff
@@ -2,12 +2,22 @@
   import { H1 } from '$lib/shared/ui';
   import { likesStore } from '$lib/features/like-episode';
   import { EpisodeCard } from '$lib/widgets/episode';
+  import { LoginContainer } from '$lib/widgets/user';
```

Comment:
issue: it sounds strange to me how a "login container" is inside a widget called "user":
* I don't really understand what is a login container
* I don't understand what could the "user" widget contain

suggestion: let's move the code that represents the user into `entities/user` and combine it with the buttons on the `pages` layer, I don't think we need widgets here